### ISSUE TITLE
Regrading is being dominated by upgrading

### ISSUE THREAD

**Issue by willmonks** [theCrag/website#2640]
User grade corrections are good but by my observation a vast majority of regrading is upwards. At the risk of sounding old and grumpy, many MANY proposed upgrades are coming from users who shall we say spend a lot of time at Bell or New Nowra and never climb on any routes from the 60-80s and who are therefore only comparing to (soft!) recent routes. 

Doing a few Ewbank 17-21s from the 1960s/70s (which, after all, is the definition of the grade) or a Giles Bradbury 20-something from the 1980s would quickly stop some people whinging about modern sport grades... but I digress. 

My point/question is whether thecrag needs some checks and balances in place to counteract the constant pressure towards grade inflation?   E.g. I wonder if this could be achieved by having a rule that each person can only propose UPgrades for the same number of routes as they have DOWNgraded (or within some proportion like 2:1).  (And for fairness the inverse rule for chronic downgraders ... not that there are any!)  <end whinge> ;)

**Comment by scd** [comment]
... and this will become a lot more important after we have made some planned enhancements.

We are aware of it and have some plans to deal with this, see floating grades - #2554 and #689 

It will have to be dealt with in the fullness of time.

**Comment by DaneEvans** [comment]
I've noticed a few recently, generally with the person in question not logging a personal grade. I certainly think it should be easier to find and change the personal contribution than the community grade, or maybe it should be locked down so only upper level users can modify the community grade. 

I like being able to change it for the first few ascents (and these people are usually talking to each other), but people making changes to 80's routes is ridiculous

**Comment by brendanheywood** [comment]
I really like the idea of peoples grade suggestions being weighted somehow according to the their previous suggestions. Almost like have a 'grade budget' of say X, and every time they make a suggestion they lose some of their budget. I probably wouldn't make them lose the ability entirely to propose a grade, I would just make each new downgrade worth progressive less and less.

**Comment by willmonks** [comment]
I think we're on the same page, but just to be clear, my thinking is that
the "budget" or "reduced valuation" idea is good if it's responsive to the
*balance* of that user's contributions, but I don't think it
should penalise the volume of contributions (I'd assume you'd want to
encourage lots of user input not discourage it). Hence my thinking that the
budget (by whatever suitable rule) should penalise an excessive *imbalance*
of upgrades vs downgrades, while being agnostic about the total volume of
regrading.

Maybe the "budget" should be inversely proportional ... e.g. if a user
suggests 100 upgrades for every downgrade, then all the upgrades are
counted only as a "0.01 vote" and each downgrade counts as a full vote.

Maybe a bit cheeky but this could even be a metric you present on the user
profile: "John is a 99% upgrader" ... if everyone can see they are a weeny
it might help keep the bastards honest.  ;)

Thanks guys as always for listening.

**Comment by Drazhar** [comment]
I like the idea that the community grade is floating and automatically adjusted depending on the personal grades, so that the community grade don't have to or even can't be adjusted. But for this, making a personal grade contribution has to be build into the logging/ticking process. If only user make a contribution who think that the grade should be different, will lead in a misjudging. Another thing to consider is, that many users just tick routes without considering that the given grade could be "wrong".

I'm absolutely no fan of something like a budget you have and you have to upgrade routes to be able to downgrade, because this makes no sense. If I'm in an area where a lot of routes are rated to high, then I downgrade them and vice versa... It's my personal opinion, so why should the system hinder me doing that? Also for the community grade. If I'm creating a crag at first hand, I also have to contribute a community grade and here I'm so free to not only use a guidebook but also my opinion if I have done the route.

And it was written that the grades in the 60s or 80s are correct, which I personally also disagree. Back in the time, the hardest grade out there was an UIAA 6 and it was considered that there couldn't be any harder routes. So a lot of 6s fron that time a really 7s or even harder. Also because of broken holds, an old route could have become significantly harder now days.

**Comment by brendanheywood** [comment]
Yeah there is definitely a lot (as always) to consider. @willmonks we are on the same page ie intent of that idea was to reduce the impact of serial up/down graders. But I'm also very conscious of the issue @Drazhar mentioned, where let's say a serial up-grader is the one who originally entered the routes, so now everyone has to go vote to downgrade them. My current thinking is something along the lines of if 10 people vote for a grade to be say 24, and a couple voted for 25, then the consensus is ~24 and only the people who voted differently have the difference (+1) taken out of their 'budget'. But then a democratic system like this will also suffer if the number of new upgraders outweigh the more considered people (which will eventually always happen). I'd consider taking karma into account too. Also possibly the grade range people are climbing at. Maybe each grade suggestion could also be partially weighted by it's age.

**Comment by DaneEvans** [comment]
+1, 
Can we at least remove the ability to unilaterally upgrade something with 130+ascents, 3 personal/guidebook grades that agree, while you have less than 500 Karma?

Even just reordering the GUI entries so the personal one is on top might be enough, because very few of the people that do this log a personal grade, so I thing it might just be confusion. 

![image](https://user-images.githubusercontent.com/22091505/29856758-47f8ad9e-8d98-11e7-9045-df693632ca82.png)
